overlays/holo-nixpkgs/default.nix

```
@@ -194,6 +194,8 @@ in
     python3 = python3.withPackages (ps: [ ps.requests ]);
   };
 
+  holo-cloudflare-wrangler = callPackage ./holo-cloudflare-wrangler {};
```

There is nothing Holo about this package, and its canonical upstream name is just `wrangler`.

```suggestion
  wrangler = callPackage ./wrangler {};
```

---

done